auditbeat/module/auditd/_meta/fields.yml

```diff
@@ -829,6 +829,7 @@
           type: keyword
           description: This is the path associated with a unix socket.
     - name: messages
+      # As it's the raw message should it be keyword. Should it even be indexed?
```

Comment:
It's rarely used (it requires a config change to enable), but when it is used it's really useful to have as text so that you can search it.

Now that we have `event.original`, I'm thinking this would make sense to put there (even if it is a keyword). Likewise the `warnings` field below could go in as `error.message`.

Reply:
This field should be keyword. If we want to enable full text search, `text` should be added as a multi-field.

Reply:
@webmat If it's called `message` it should be text by default. So far this convention holds in ECS I think.

Reply:
@andrewkroh I assume your team will tackle these changes?